spec/slack-ruby-bot-server-events/api/endpoints/slack/actions_endpoint_spec.rb

```diff
@@ -48,6 +48,19 @@
         end
       end
 
+      context 'without actions in the payload' do
+        before do
+          payload.delete(:actions)
+        end
+
+        it 'performs action' do
+          post '/api/slack/action', payload: payload.to_json
+          expect(last_response.status).to eq 201
+          response = JSON.parse(last_response.body)
+          expect(response).to eq('text' => 'message_action: type match')
```

Comment:
You need a spec that does an action, otherwise this is failing predictably to return any data.